antfs_cli/utilities.py

```diff
@@ -38,6 +38,7 @@ def makedirs_if_not_exists(path):
 
 class XDGError(Exception):
     def __init__(self, message):
+        super(XDGError, self).__init__(message)
         self.message = message
```

Comment:
Change made.
As the `message` is just being passed up to the super class and not actually used in `XDGError()`, I think this is fine.